PlantPal/PlantPal/View/ReusableView/PalDivider.swift

```diff
@@ -8,14 +8,22 @@
 import SwiftUI
 
 struct PalDivider: View {
+    
+    enum divaiderColor: String {
+        case mainView = "DividerColor"
+        case welcomeScreen = "DividerMainScreen"
+    }
+    
+    var color: divaiderColor
+    
     var body: some View {
         Rectangle()
-             .fill(Color(.divider))
+            .fill(Color(color.rawValue))
              .frame(height: 1) // Ustawiamy tylko wysokość
```

Comment:
Niepotrzebny ten komentarz. Z komentarzami to jest tak, że powinniśmy ich unikać bo sam kod powinien być samoopisujący się. A nawet jeżeli zdecydujemy się napisać komentarz to bardziej nie co robi, a z czego wynika taki sposób użycia. I zdecydowanie unikamy polskiego jezyka - English for the win!